style.css

```diff
@@ -289,22 +260,19 @@ header {
   top: calc (50% - 48px/2);
   border: 1px solid #6070ff;
   border-radius: 8px;
-}
-
-.see {
```

Comment:
- Kindly,  most of the buttons on your design don't have hover and pressed (active) interactions on desktop designs.👌👍 Kindly implement this feature as it is a requirement in this project.👍
- The screenshot below shows the section where you can get the recommended design for the button interactions👌👍
Kindly implement this feature for the See Project button on all cards, the Get My Resume button, and the Form Button as the previous code reviewer suggested you 👌👍
<img width="611" alt="Screenshot 2023-01-27 at 11 53 06 AM" src="https://user-images.githubusercontent.com/52010789/215046311-0a1d24de-3dad-4d5d-b016-0aada8bb31d7.png">